### PR TITLE
Add support for a couple ops

### PR DESCRIPTION
**Pull request by eellison**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #79809
* __->__ #79581

### CODE REVIEW

File: torch/_subclasses/fake_tensor.py [hunk 1]
```diff
@@ -442,18 +461,25 @@ def from_tensor(self, tensor):
...
             if isinstance(e, FakeTensor):
-                return torch.zeros_like(e, device="cpu")
+                return torch.ones_like(e, device="cpu")
             return e
 
         try:
             args = tree_map(to_cpu, args)
             kwargs = tree_map(to_cpu, kwargs)
 
+            # TODO: full decomposition/fix
```
Comment (Chillee) [inline torch/_subclasses/fake_tensor.py:475]: Add note that this is a hack due to no meta coverage. But isn't there already a meta-function for `aten._embedding_bag`?

Comment (eellison) [reply torch/_subclasses/fake_tensor.py:475]: There's not, there is `_embedding_bag_forward_only`

File: torch/_refs/__init__.py [hunk 2]
```diff
@@ -1102,7 +1102,7 @@ def clamp(
 
 # https://pytorch.org/docs/stable/generated/torch.where.html
 # TODO: implement alternate where
-@register_decomposition(torch.ops.aten.where)
+@register_decomposition(torch.ops.aten.where, disable_meta=True)
```
Comment (Chillee) [inline torch/_refs/__init__.py:1105]: just modify this to `torch.ops.aten.where.self`

**Review by Chillee — APPROVED** [review]
LGTM I suppose, modulo one comment.

**Pull request closed without merging**